tests/installation/welcome.pm

```diff
@@ -71,7 +71,6 @@ sub get_product_shortcuts {
             sled => 'x',
             sles4sap => is_ppc64le() ? 'i'
             : (is_sle('15-SP2+') && is_x86_64() && !is_quarterly_iso()) ? 't'
-            : (is_sle('15-SP3+') && is_x86_64()) ? 't'
```

Comment:
I think that this removal will break 15-SP4 tests, as `t` is still use as a link for 15-SP4: https://openqa.suse.de/tests/7126124#step/welcome/5.

And this is because Real Time version of SLE is not part of the current beta stage, it will be added after GM. That's why we have the issue now in 15-SP3.

Reply:
Good point, let me add the line for 15-SP4.